### PR TITLE
add evaluation code and ncnn module

### PR DESCRIPTION
**Pull request by vealocia**
this is the new version which has fixed some bugs in #29 

> this pull request convolved my implementation of widerface evaluation generator code and ncnn c++ implementation. you can merged into master after a carefully check. wait for your reply

> sorry. I found some wrong code in my code and this code may caught an error in Android deployment. I'll close this pull request until i fixed it and create a new one.

### CODE REVIEW

**Comment by ingjieye** [comment]
I've tested your code but the bonding box seems not quiet working well.

**Comment by ingjieye** [comment]
I've tested your code but the bonding box `UltraFace` gives seems not working quiet well. It bias too much from the python version.

**Comment by vealocia** [comment]
get it. I'll check it later.

**Comment by Linzaer** [comment]
I tested it and found something wrong with the face boxes. I will check it too.

File: ncnn/UltraFace.cpp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        case 128:{
+            in_w = 128;
+            in_h = 96;
+            num_anchors = 708;
+            break;
+        }
+        case 320:{
+            in_w = 320;
+            in_h = 240;
+            num_anchors = 4402;
```
[the remark this answers is not in the document]
Comment (ingjieye) [reply ncnn/UltraFace.cpp]: By changing the `num_anchors` to 4420 did improve the result `bounding box` a little bit. But still bias too much from the python version.

[the remark this answers is not in the document]
Comment (Linzaer) [reply ncnn/UltraFace.cpp]: @ingjieye Yes. I think it's possible that the operation of 'convert_locations_to_boxes()' function has an little error in NCNN . I'll comment out this line of code in ssd.py and export to onnx then try again.

**Pull request closed without merging**